src/main.js

```diff
@@ -153,7 +138,7 @@ function makeNew() {
 function makeMyBook (){
   event.preventDefault();
   if (!userCoverField.value || !userTitleField.value || !userTagline1Field.value || !userTagline2Field.value) {
-    createNewBookButton.disabled = true;
+    createNewBookButton.disable = true;
```

Comment:
I believe you _might_ have this right the first time? [Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled)

Reply:
Yes, disabled.